lib/src/requester/node_cache.dart

```diff
@@ -27,6 +27,8 @@ class RemoteNodeCache {
 
   Iterable<String> get cachedNodePaths => _nodes.keys;
 
+  int get count => _nodes.length;
```

Comment:
I believe `.length` would feel more darty

Reply:
I'll change that. I agree it makes more sense.